src/ngx_http_lua_ssl_session_fetchby.c

```diff
@@ -451,7 +451,7 @@ ngx_http_lua_log_ssl_sess_fetch_error(ngx_log_t *log, u_char *buf, size_t len)
 
         if (c->listening && c->listening->addr_text.len) {
             p = ngx_snprintf(buf, len, ", server: %V", \
-                &c->listening->addr_text);
+            &c->listening->addr_text);
```

Comment:
The `&c->...` should have the same indentation like the `buf` above:
```
p = ngx_snprintf(buf, len, ", server: %V", \
                 &c->listening->addr_text);
```

Reply:
fixed